HeavyIonsAnalysis/JetAnalysis/python/inclusiveJetAnalyzer_cff.py

```diff
@@ -22,7 +22,8 @@
                                       doHiJetID = cms.untracked.bool(True),
                                       doStandardJetID = cms.untracked.bool(False),
                                       doSubEvent = cms.untracked.bool(False),
-                                      
+	                              jetIDWeight = cms.untracked.string("HeavyIonsAnalysis/JetAnalysis/data/JetID_TMVAClassification_BDTG_weights.xml"),       
```

Comment:
using the edm::FileInPath methodology in line 190 of the .cc makes the path visible to both crab and interactive jobs so i think we're ok now